addons/account/views/account_move_views.xml

```diff
@@ -1146,8 +1146,7 @@
                                                options="{'product_field': 'product_id', 'account_field': 'account_id', 'amount_field': 'price_subtotal'}"
                                                business_domain_compute="parent.move_type in ['out_invoice', 'out_refund', 'out_receipt'] and 'invoice' or parent.move_type in ['in_invoice', 'in_refund', 'in_receipt'] and 'bill' or 'general'"/>
                                         <field name="quantity" optional="show"/>
-                                        <field name="product_uom_category_id" column_invisible="True"/>
-                                        <field name="product_uom_id" string="UoM" groups="uom.group_uom" optional="show" width="92px"/>
+                                        <field name="product_uom_id" string="Unit" groups="uom.group_uom" optional="show" width="92px" options="{'no_create': True}" widget="many2one_uom"/>
```

Comment:
You won't need the string override here anymore now that you changed the field string :wink: